src/api/graphql.ts

```diff
@@ -101,7 +105,12 @@ export async function getDeployedContractsByWallet(
 
 export async function getWalletInteractionsForContract(
   arweave: Arweave,
-  params: { address?: string; contractTxId: string },
+  params: {
+    address?: string | undefined;
+    contractTxId: string;
+    sortKey: string | undefined;
+    blockHeight: number | undefined;
```

Comment:
is it necessary to have the `?` on `address` if we're directly declaring undefined in the union type?

Reply:
they mean different things - `?` means you do not have to provide the param. | undefined means you can provide it and it's value can be undefined